frontend-react/src/styles/_components.scss

```diff
@@ -115,4 +115,3 @@ hr {
         margin-top: 40px;
     }
 }
-
```

Comment:
Reverting this change should hopefully prevent the chromatic workflow from running.